mlflow/models/utils.py

```diff
@@ -68,7 +68,7 @@ def _handle_tensor_input(input_tensor: Union[np.ndarray, dict]):
                     result[name] = input_tensor[name].tolist()
                 return {"inputs": result}
             else:
-                return {"instances": input_example.tolist()}
+                return {"inputs": input_example.tolist()}
```

Comment:
oops sorry missed this
```suggestion
                return {"inputs": input_tensor.tolist()}
```

Reply:
nice catch Wendy! I guess the both work in this case cause they are the same?

Reply:
yeah and `input_example` doesn't get modified anywhere in the constructor so it technically doesn't matter, but felt dangerous to leave as is